syntax = "proto3";

package client;

message ClEngineMakeMove {
  int32 id = 1;
  string move = 2;
}